lib/samples/identify_layer_features/README.md

```diff
@@ -0,0 +1,35 @@
+# Identify layer features
+
+Identify features in all layers in a map.
+
+![Image of identify layer features](identify_layer_features.png)
+
+## Use case
+
+`identifyLayers` allows users to tap on a map, returning features at that location across multiple layers. Because some layer types have sublayers, the sample recursively counts results for sublayers within each layer.
+
+## How to use the sample
+
+Tap to identify features. A banner will list layers with features under the tapped location, as well as a layer count.
```

Comment:
This deviates from the design, though I know Swift deviated, in showing a banner rather than a dialog. I don't really mind, but it'll probably be one to note at verification as a behavior difference.